examples/simple_fbs_with_socs.ipynb

```diff
@@ -0,0 +1,215 @@
+{
+ "cells": [
```

Comment:
These notebooks are great!  I usually run them and commit with the results so one can see the output on github (makes it easy to compare if you get the right results when you run it yourself).

One thing that would be nice to add is to show where the executed observations are. That's one nice part of `sim_runner` is that it also hands back an array with all the observations. 

This does seem to take a while to get through all the initialization steps. Would be good to find the startup bottlenecks. When I do 1 night with `sim_runner` it executes in 80 seconds while 1 night with `sim.run()` takes 11 minutes. 

The examples directory is probably due for some cleaning, I'm sure plenty of the older ones are bit rotted. It might make sense to have a new repo with a set of example simulation configurations. It can be useful for debugging and teaching others to have things like a 1-filter survey, etc.